features/FxA-51-connect-another-device/README.md

```diff
@@ -0,0 +1,130 @@
+
+We need to better guide the user through the setup of two devices while removing as much friction as possible. Next steps should always be clear to users.
+
+### Assumptions
+We assume that users signing up for Sync have an understanding of why they are setting up an account so we should merely be facilitating setup after email confirmation.
+
+(edit: we have observed through user testing that new users do not understand the first run page very well. This will be the subject of one of our next tests.)
+
+## Outcomes
+The outcome we are looking for is a slick user flow that makes it obvious to users that in order to complete their setup of Sync, they need to add another device after email confirmation.
+
+## Hypothesis
+If we refine our UI/UX during sign up to remove friction around setting up a second device for firstrun users on desktop, then we will see an increase in multi-device users within the first 7 days of registration because we have observed that less than 30% of registration (65k clicks / 210k registration / month) engage with our mobile download buttons and at best 26% (8% 7d md / 30% ctr) of them follow through to an install within 7 days after clicking.
+
+## Metrics
+We will know our hypothesis to be true when we see the proportion of multi-device users increase after 48h and 7d of registration to FxA+Sync on firstrun.
+
+![signups that connect multiple devices](signups-connect-multiple-devices.png)
+From: https://sql.telemetry.mozilla.org/queries/1254#2173
+
+We will also want to keep an eye out for:
+Email confirmation rate: we should not impact this negatively in any way.
+Measured with flow events and visualized in re:dash
+Install rate from confirmation page on mobile: If users confirmed their email on mobile, we will drive users to install via smart banner. (install rate)
+
+
+### Additional AB metrics:
+% of users that verify in a second Firefox instance (Desktop or Fennec) that connect that instance. XXX stomlinson to look up existing metrics to see if any match.
+% of users that see the app store links that click on a link. XXX we log marketing impressions, and clicks and send this data to GA. stomlinson to figure out how to do this as part of flow events too.
+
+
+
+## Detailed design
+### Audience
+This test will target:
+EN-* countries
+New account users signing up to Sync (first run, preferences, etc)
+
+
+Here is the flow of the current version:
+
+![Current Sync onboarding experience](current-sync-onboarding.png)
+
+### Variation A
+Here is the user flow from the moment the user finishes registration via the form (in dashed outline box).
+
+![Proposed Sync onboarding experience](proposed-sync-onboarding.png)
+
+And here are the screenshots
+
+1. User is asked to confirm email address
+
+![Confirm your account screen](confirm-your-account.png)
+
+Email prepares the user for next steps
+
+![Updated sync specific email](ready-set-sync-email.png)
```

Comment:
might be useful to use `img` tags here to make this a bit smaller so it is easier to read the doc